### PR TITLE
Story/quad77

### PR DESCRIPTION
**Pull request by rmanojcit**
autocrlf and merge with develop

### CODE REVIEW

File: Quadriga/.classpath [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+			<attribute name="org.eclipse.jst.component.dependency" value="/WEB-INF/lib"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7">
+		<attributes>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="output" path="target/classes"/>
+</classpath>
```
Comment (jdamerow) [inline Quadriga/.classpath:30]: You still seem to submit this file with the wrong line endings.

File: Quadriga/.settings/org.eclipse.wst.common.component [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+        <wb-resource deploy-path="/" source-path="/target/m2e-wtp/web-resources"/>
+		<wb-resource deploy-path="/" source-path="/src/main/webapp" tag="defaultRootSource"/>
+        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/java"/>
+        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/test/java"/>
+        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/resources"/>
+        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/test/resources"/>
+		<property name="java-output-path" value="target/classes"/>
+        <property name="context-root" value="quadriga"/>
+	</wb-module>
+</project-modules>
```
Comment (jdamerow) [inline Quadriga/.settings/org.eclipse.wst.common.component:12]: same here?

File: Quadriga/src/main/webapp/WEB-INF/views/editing/networkvisual.jsp [hunk 3]
```diff
@@ -34,7 +34,7 @@ $("input[type=button]").button().click(function(event) {
 
 <body
 	onload="d3visualize(<c:out value='${jsonstring}'></c:out>,<c:out value='${networkid}'></c:out>,<c:out value='"${pageContext.servletContext.contextPath}"'></c:out>,'force');" />
-
+<input type="submit" onclick="goFullscreen('chart')"  class="fa fa-2x" style="float: left" value="&#xf0b2;"/>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/editing/networkvisual.jsp]: usually the font awesome class is in it's own span or i element.

**Comment by rmanojcit** [comment]
sending a new pull request

**Pull request closed without merging**